.changelog/2985.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:bug
```

Comment:
Is this a bug? Should we call it as an enhancement?

Reply:
reported as a bug, I too had this confusion.

Reply:
It could be an improvement

Reply:
I'd call it an 'improvement' also.

Reply:
updated